### PR TITLE
[CRBFUS-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by IreneNakiyingi**

### CODE REVIEW

File: submissions/ireneNakiyingi/HTML/bakery.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>FLUFF BAKES</title>
+        <nav>
+            <a href="bakery.html">Home</a>
+            <a href="cakes.html">Cakes</a>
+            FLUFF BAKES 🎂🍪🍩🥨
+            <a href="bread.html">Bread</a>
+            <a href="contact.html">Contact</a>
```
Comment (Ireti203) [inline submissions/ireneNakiyingi/HTML/bakery.html:10]: Are you sure this link works? @IreneNakiyingi

Comment (Ireti203) [inline submissions/ireneNakiyingi/HTML/bakery.html:10]: Also, do all these pages exist? You ought to have only two pages for now. The contact page and the product page

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by ToluEnny24** [comment]
@IreneNakiyingi what blockers have caused this delay in your pr?

**Comment by IreneNakiyingi** [comment]
> @IreneNakiyingi what blockers have caused this delay in your pr?

Hello,  I made three pull requests in this module.  One had an error in the naming (wrong names to the wrong files) and I was told to make another. But it was never merged. Logfile Json failed to be added by me.

**Review by ToluEnny24 — APPROVED** [review]
Manually reviewed..

**Pull request merged**